classes/DirtOrbit.sc

```diff
@@ -173,22 +175,24 @@ DirtOrbit {
 			~overgain = 0.0;
 			~cut = 0.0;
 			~unit = \r;
-			~n = \none; // sample number or note
-			~octave = 5;
-			~midinote = #{ ~note ? ~n + (~octave * 12) };
+			~n = nil;
+			~octave = 0;
+			~midinote = #{ (~note ? ~n ? StrudelUtils.baseNote()) + (~octave * 12) };
 			~freq = #{ ~midinote.value.midicps };
 			~dur = 1.0;
 			~delta = #{ ~dur.value };
-
 			~latency = 0.0;
 			~lag = 0.0;
 			~length = 1.0;
 			~loop = 1.0;
 			~dry = 0.0;
 			~lock = 0; // if set to 1, syncs delay times with cps
 
-			~amp = 0.4;
+			~amp = 1;
 			~fadeTime = 0.001;
+			~delaytime = 0.1875;
```

Comment:
I am not 100% sure, but I think this means that there is always delay on. This can be set for each piece separately.